subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -258,9 +258,53 @@ This message means that either
 --
 In both cases the build cannot benefit from file system watching.
 
-`Caught exception: Couldn't add watch, error = 28`::
-If you receive this error on Linux then you ran out of inotify handles.
-To raise the limit see https://confluence.jetbrains.com/display/IDEADEV/Inotify+Watches+Limit[here].
-
 `java.io.IOException: Too many open files`::
 If you receive this error on macOS, you need to raise your open files limit, see https://superuser.com/a/443168/8117[here].
+
+===  Linux specific notes
+
+File system watching uses http://en.wikipedia.org/wiki/Inotify[Inotify] on Linux.
```

Comment:
The word `inotify` is all lower-case, even in the title of the Wikipedia page, I think that's how we should use it here, too (unless at the start of a sentence). If we want to highlight it, maybe using a monospace font would be appropriate.

Reply:
I changed it all to lower-case.